aptos/Dockerfile

```
@@ -1,3 +1,9 @@
+FROM node:lts-alpine@sha256:2ae9624a39ce437e7f58931a5747fdc60224c6e40f8980db90728de58e22af7c as worm-cli
+
+COPY clients/js  /tmp/clients/js
+WORKDIR /tmp/clients/js
+RUN make install
```

Your code will work, but docker won't cache it too well. Docker's caching works by looking at the files you copy, and if nothing has changed, then the following commands are not re-evaluated, just brought in from cache. However, if a single file changes, then the whole cache gets invalidated. In concrete terms, that means that if any of the typescript files change in `clients/js`, then `make install` will start over with no cache, and will reinstall all the npm dependencies again.

A better way is to make the file copy a little more granular, start with `package.json` and `package-lock.json`, and run `npm ci` first, and only then invoke `make install`. This will ensure that dependencies are only installed when the package manifest files actually change (then `make` is clever enough to not run `npm ci` again). Also, when running `npm ci` we can specify a cache mount, which basically tells docker to keep that folder from previous invocations of this command.
```suggestion
WORKDIR /tmp/clients/js
COPY clients/js/package.json clients/js/package-lock.json ./
RUN --mount=type=cache,target=/home/node/.npm \
    npm ci
COPY clients/js ./
RUN make install
```